test/scripts/e2e_subs/rest-participation-key.sh

```diff
@@ -9,33 +9,48 @@ date "+$0 start %Y%m%d_%H%M%S"
 # Use admin token for both get and post
 export USE_ADMIN=true
 
-pushd "${TEMPDIR}" || exit
+pushd "${TEMPDIR}" || exit 1
 
 FIRST_ROUND=0
 # A really large (but arbitrary) last valid round
-LAST_ROUND=1200000
+LAST_ROUND=120
 
 NAME_OF_TEMP_PARTKEY="tmp.${FIRST_ROUND}.${LAST_ROUND}.partkey"
 
 algokey part generate --first ${FIRST_ROUND} --last ${LAST_ROUND} --keyfile ${NAME_OF_TEMP_PARTKEY} --parent ${ACCOUNT}
 
-popd || exit
+popd || exit 1
 
 call_and_verify "Get List of Keys" "/v2/participation" 200 'address'
 
 # Find out how many keys there are installed so far
+if [[ $RES != *"address"*"effective-first-valid"* ]]; then
```

Comment:
`call_and_verify` accepts a vararg for patterns to compare. So you should be able to add `effective-first-valid` there instead of having this check (and some of the checks below).